quipucords/tests/report_utils.py

```diff
@@ -0,0 +1,39 @@
+"""Utility functions to help tests for API reports."""
+
+import tarfile
+from io import BytesIO
+from pathlib import Path
+
+from requests import Response
+
+
+def extract_tarball_from_response(
+    response: Response, strip_dirs: bool = True
```

Comment:
instead of expecting a response we could expect `bytes`, to make this function more generic.

Reply:
Yeah, I'll do that so I can use it to replace `extract_tar_gz`. 👍

Reply:
Done!